tests/testthat/test-growth_rate.R

```diff
@@ -0,0 +1,14 @@
+library(dplyr)
+
+x <- jhu_csse_daily_subset %>%
+  select(-case_rate_7d_av,-death_rate_7d_av, -cases_7d_av) %>%
+  group_by(geo_value) %>% 
+  mutate(cases_gr = growth_rate(x = time_value,  y = cases))
```

Comment:
Looks like this `x` isn't used, and might cause confusion win the `x` columns below.  Maybe remove this statement, or put it inside an `expect_error(jhu_csse_daily_subset %>% <...>, NA)`?